src/main/java/com/suse/saltstack/netapi/datatypes/Token.java

```diff
@@ -31,4 +34,8 @@ public Date getStart() {
     public Date getExpire() {
         return expire;
     }
+
+    public List<String> getPerms() {
```

Comment:
No documentation on the public method. I recon, you wanted to do something with permissions?